Tests/Stubs/Menu/ContainerAwareBuilder.php

```diff
@@ -3,10 +3,18 @@
 namespace Knp\Bundle\MenuBundle\Tests\Stubs\Menu;
 
 use Knp\Menu\FactoryInterface;
-use Symfony\Component\DependencyInjection\ContainerAware;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 
-class ContainerAwareBuilder extends ContainerAware
+class ContainerAwareBuilder implements ContainerAwareInterface
```

Comment:
From my understanding of the changes coming this does not get it ready for Symfony 3.0. ContainerAwareInterface is what is deprecated. You need to `use ContainerAwareTrait;` is the recommended fix.

Thoughts? @WouterJ & @stof